tests/test_swf.py

```diff
@@ -71,6 +71,14 @@ def test_reading_saving():
     W.set_meta_data({"foo": 3})
     W.close()
 
+    # Just make sure mimread works
+    assert len(imageio.mimread(fname1)) == 10
+
+    # I'm not sure why, but the below does not work on pypy, which is weird,
+    # because the file *is* closed, but somehow it's not flushed? Ah well ...
+    if IS_PYPY:
+        return
```

Comment:
I guess we can leave this for the current PR (doesn't seem like a high priority), but could you create an issue referencing this, so that we can look into and fix this test later?

Reply:
https://github.com/imageio/imageio/issues/598  - I did not spend too much time on this, since SWF is not very useful anymore since Flash is out of fashion. But it would be worth verifying whether this issue is specific to SWF or affects all multi-image writing.